sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/catalog/SessionCatalog.scala

```diff
@@ -693,7 +697,6 @@ class SessionCatalog(
       dropDatabase(db, ignoreIfNotExists = false, cascade = true)
     }
     tempTables.clear()
-    functionRegistry.clear()
```

Comment:
Is this related to the `current_database()` issue you and @yhuai discussed previously?